File: URI_1113.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
        int x, y ;

		while (1) {
			cin>>x;
			cin>>y;

			if (x == y)
				break;

			else if (x > y) {
				cout<<"Decrescente"<<endl;
			} else
				cout<<"Crescente"<<endl;
		}
 
    return 0;
}

File: URI_1113.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner scanner = new Scanner(System.in);
		int x, y, temp, sum;

		while (true) {
			x = scanner.nextInt();
			y = scanner.nextInt();

			if (x == y)
				break;

			else if (x > y) {
				System.out.println("Decrescente");
			} else
				System.out.println("Crescente");
		}
 
    }
 
}
